### ISSUE TITLE
Remove trailing '?' on path `/toolkit/`

### ISSUE THREAD

**Issue by 8alpreet** [hackforla/website#7636]
### Overview

We need to remove the default trailing '?' at the end of the `/toolkit/` path so that anchor links will work on the page.

### Action Items

- [x] Find out why the Toolkit page URL is `/toolkit/?` instead of `/toolkit/` by default
- [x] Make the necessary code changes
- [x] Test filtering functionality
    - [x] Does filtering work?
    - [x] Is there a '?' at the beginning of the filters in URL when at at least on filter is checked?
    - [x] Are filters added to the URL when checked?
    - [x] Are filters removed from the URL when unchecked?
    - [x] After checking a filter then unchecking it, does the '?' once again disappear?
- [x] There should be no visual changes to the website except the URL. 
- [x] Release dependency on #7668

### Resources/Instructions
- This issue resulted from #6919 
- [Info about source of '?'](https://github.com/hackforla/website/issues/6919#issuecomment-2438819611)

**Comment by 8alpreet** [comment]
Product team, can I proceed with this?

**Comment by 8alpreet** [comment]
The ER has `role: back end/devops` but this is a frontend issue. So we should either have only `role: frontend` or both.

**Comment by ExperimentsInHonesty** [comment]
@8alpreet Please rewrite the title and the overview

Title should be short and more focused, e.g. Research XYZ 
Overview should be in the following format so that we can control scope (we need to do X for Y reason)

**Comment by HackforLABot** [comment]
Hi @8alpreet, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by ExperimentsInHonesty** [comment]
This person is no longer active.  Product to review the issue draft and see if its ready or rewrite it.

**Comment by HackforLABot** [comment]
Hello @Six5pAdes, we appreciate you taking on this issue, however it looks like you're already working on another issue at this time. Please wait until your current issue is merged before taking on another issue. :)

We are going to unassign you from this issue so you can focus on your current issue.

Hfla appreciates you! :)

**Comment by HackforLABot** [comment]
Hi @eunicode, thank you for taking up this issue! Hfla appreciates you :)

Do let fellow developers know about your:-
i. **Availability:** (When are you available to work on the issue/answer questions other programmers might have about your issue?)
ii. **ETA:** (When do you expect this issue to be completed?)

You're awesome!

P.S. - You may not take up another issue until this issue gets merged (or closed). Thanks again :)

**Comment by eunicode** [comment]
i. Availability: Mon-Sat, 5pm - 11pm PDT

ii. ETA: Completed. 

Once the PR is merged, a maintainer will have to release dependency on #7668
